packages/frontend/apps/electron-renderer/src/setup.ts

```diff
@@ -1,3 +1,9 @@
 import '@affine/core/bootstrap/electron';
 import '@affine/component/theme';
 import './global.css';
+
+import { apis } from '@affine/electron-api';
+import { bindNativeDBApis } from '@affine/nbstore/sqlite';
+
+// oxlint-disable-next-line no-non-null-assertion
+bindNativeDBApis(apis!.nbstore);
```

Comment:
what's this for?

Reply:
some places in the main thread will directly use `new SqliteDocStorage(...)`.